src/java.base/share/classes/java/lang/String.java

```diff
@@ -248,8 +249,10 @@
      * an empty character sequence.  Note that use of this constructor is
      * unnecessary since Strings are immutable.
      */
+
     @SideEffectFree
     @StaticallyExecutable
+    @Unique
```

Comment:
Put type use annotations before the types they modify. Here, after the existing `@StringVal` to sort them alphabetically.

Reply:
I have changed the annotation location (in alphabetical order) in my latest commit